configs/edge-config.yaml

```diff
@@ -14,3 +14,4 @@ detectors:
       local_inference_template: "default"
       always_return_edge_prediction: false
       disable_cloud_escalation: false
+      min_time_between_escalations: None
```

Comment:
I think it would be good to add some mention of this to the README. The READMEs need to get reorganized at some point so it doesn't matter too much where it goes, but having a mention of this config option in there somewhere would help make sure it makes it into the reworked version too.

Reply:
Good idea! I'll add it.